### PR TITLE
Baozi: add new domain

### PR DESCRIPTION
**Pull request by ifurther**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [ ] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

**Comment by AnonymousPtBr** [comment]
👍

**Review by AwkwardPeak7 — CHANGES_REQUESTED** [review]
you need to bump the version

File: src/zh/baozimanhua/src/eu/kanade/tachiyomi/extension/zh/baozimanhua/Baozi.kt [hunk 1]
```diff
@@ -290,6 +290,7 @@ class Baozi : ParsedHttpSource(), ConfigurableSource {
             "cn.czmanga.com",
             "tw.czmanga.com",
             "www.czmanga.com",
+            "www.twmanga.com",
```
Comment (stevenyomi) [inline src/zh/baozimanhua/src/eu/kanade/tachiyomi/extension/zh/baozimanhua/Baozi.kt]:
```suggestion
            "cn.twmanga.com",
            "tw.twmanga.com",
            "www.twmanga.com",
```

`czmanga.com` now redirects to the new domain.

File: src/zh/baozimanhua/AndroidManifest.xml [hunk 2]
```diff
@@ -18,6 +18,18 @@
                 <data android:host="*.webmota.com" />
                 <data android:host="webmota.com" />
 
+                <data android:host="*.kukuc.co" />
+                <data android:host="kukuc.co" />
+
+                <data android:host="*.czmanga.com" />
+                <data android:host="czmanga.com" />
```
Comment (stevenyomi) [inline src/zh/baozimanhua/AndroidManifest.xml]:
```suggestion
```

File: src/zh/baozimanhua/src/eu/kanade/tachiyomi/extension/zh/baozimanhua/Baozi.kt (hunk 1 above)
Comment (ifurther) [reply src/zh/baozimanhua/src/eu/kanade/tachiyomi/extension/zh/baozimanhua/Baozi.kt]: Ok, I remove it

**Review by stevenyomi — APPROVED** [review]

**Pull request merged**